convert/README.md

```diff
@@ -93,27 +94,34 @@ Command:
 
 ### Create terraform variables
 
-First, we must create the variables going into our Terraform config.
-
-* The following command will look at the current CDK stack and autopropagate the Terraform variables for the appropriate modules by generating multiple files with extension `*.tfvars.json` under `$DEPLOY_ID/terraform`.
+Running the command below will inspect the existing CDK stack and automatically populate the corresponding Terraform variables for each module. This will generate multiple *.tfvars.json files within the `$DEPLOY_ID/terraform` directory:
 
 * `$DEPLOY_ID/terraform/cdk_tf.tfvars.json`
 * `$DEPLOY_ID/terraform/infra.tfvars.json`
 * `$DEPLOY_ID/terraform/cluster.tfvars.json`
 * `$DEPLOY_ID/terraform/nodes.tfvars.json`
 
+If the `tfvar` binary is already installed, this process will also attempt to convert the .tfvars.json files into HashiCorp Configuration Language (HCL) format for easier editing. The resulting .tfvars files will be saved in the same directory:
```

Comment:
I think if we're going to do this, it's better to just go all the way and require tfvar so there isn't this variability within the instructions.

Reply:
Done